sentinel-dashboard/src/main/java/com/alibaba/csp/sentinel/dashboard/controller/AuthController.java

```diff
@@ -77,4 +82,16 @@
         request.getSession().invalidate();
         return Result.ofSuccess(null);
     }
+
+    @RequestMapping(value = "/check")
```

Comment:
Here "/check" and "/logout" `@RequestMapping`, we can optimize them to `@PostMapping`.

Reply:
Very right!

Reply:
Thanks for the guidance, I fixed , please code review, if there is no problem, we can resolve this conversation.